src/main/java/com/minecolonies/coremod/commands/CommandEntryPointNew.java

```diff
@@ -420,18 +432,29 @@ private ParsingResult getTabCompletionsAndParsingHolders(
                 if (childs.containsKey(lowerCaseArg0))
                 {
                     final TreeNode<IMenu> childTreeNode = childs.get(lowerCaseArg0);
-                    return new ParsingResult(tabCompletions, childTreeNode, Collections.emptyList(), (String) null);
+                    final IMenu childMenu = childTreeNode.getData();
+                    final ActionMenuState actionMenuState;
+                    if (!childMenu.getMenuType().isNavigationMenu())
+                    {
+                        final ActionMenu actionMenu = (ActionMenu) childMenu;
+                        actionMenuState = new ActionMenuState(actionMenu);
+                    }
+                    else
+                    {
+                        actionMenuState = (ActionMenuState) null;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unnecessary cast to "ActionMenuState". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1905)